package.json

```diff
@@ -50,6 +50,7 @@
     "jest-date-mock": "^1.0.7",
     "lint-staged": "^9.5.0",
     "prettier": "^1.19.1",
+    "rimraf": "^3.0.0",
```

Comment:
I added this using `yarn add --dev rimraf`, but the `yarn.lock` file had no changes :O

Reply:
It's possible it was already included as a transitive dependency elsewhere.